README.md

```diff
@@ -1,4 +1,7 @@
-![Screenshot of the main page](./screenshot.jpg)
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://i.imgur.com/o7Oj6Rg.png">
+  <img alt="A quick preview of our UI" src="https://i.imgur.com/79FolWw.png">
```

Comment:
@BracketJohn what's your take? Should we always keep images directly to the repo and potentially bloat its size over time? Or maybe use lfs for them (although I don't know if lfs will be displayed in the readme)?

Reply:
Agree, we should not rely on external hosting of our images:
- may be purged accidentally (I don't think 1 view every six month is a guarantee, rather a policy)
- outage of external provider will result in (partial) outage of our service
- external provider may apply any sort of alteration to terms & conditions at any time (e.g.: injecting some sort of user tracking into image serving?)

We already have a lot of parties we have to blindly trust (AWS, github, drone, google, ...) and should be careful with adding another one.

Especially if our problem is already solved via `git-lfs`. 

> @BracketJohn what's your take? Should we always keep images directly to the repo and potentially bloat its size over time? Or maybe use lfs for them (although I don't know if lfs will be displayed in the readme)?

It's possible to reference git lfs images in readmes or elsewhere: https://stackoverflow.com/a/47371057

My take: It's fine to add images to our repos, you can decide when to introduce `lfs` to make it more efficient. Here we are talking about 305kb, for me this would be too early to introduce lfs.